components/svgs/FacebookIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { SVGProps } from "react";
+import Link from "next/link";
+
+export const FacebookIcon = (props: SVGProps<SVGSVGElement>) => (
+  <Link href="https://www.facebook.com/ICAF.org" target="_blank" rel="noopener noreferrer">
```

Comment:
These components should just encapsulate the SVG icons and not be wrapped by a link. This way the component can be used inside a link or as a decorative element in some other case. 

I think the alt text is good, but I don't think baking the alternative text into the component itself is a good idea. Alt text should ideally be declared as a prop to the component. Of course, the approaches I sent don't provide a means for doing this.

Perhaps try the "Visual but accessible icon" approach outlined in this article: https://a11y-101.com/development/icons-and-links. You should be able to remove all alt text related content from the SVGs using this approach. Feel free to turn this approach into a component...perhaps making a component `IconLink` that works like this: 

```
<IconLink 
    altText="International Child Art Foundation's Facebook page" 
    href="https://www.facebook.com/ICAF.org" 
    target="_blank"
    rel="noopener noreferrer"
>
    <FacebookIcon />
</>
```

Reply:
that makes a lot of sense